app/src/main/java/org/luxoft/sdl_core/IniLoader.java

```diff
@@ -0,0 +1,112 @@
+package org.luxoft.sdl_core;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class IniLoader {
+
+    private HashMap<String, HashMap<String, String>> mDataMap;
+    private String mFilePath;
+    private boolean mIsLoaded = false;
+
+    public boolean load(String filePath) {
+        mFilePath = filePath;
+        return loadProcess(filePath);
+    }
+
+    public boolean reload() {
+        return isEmpty(mFilePath) && loadProcess(mFilePath);
+    }
+
+    private boolean loadProcess(String filePath) {
+        mIsLoaded = false;
+        mDataMap = new HashMap<>();
+        try {
+            FileReader fileReader = new FileReader(new File(filePath));
+            BufferedReader br = new BufferedReader(fileReader);
+            String line = br.readLine();
+
+            String section = null;
+            HashMap<String, String> map = new HashMap<>();
+            while (line != null) {
+                line = line.trim();
+
+                //Blank line
+                if (isEmpty(line)) {
+                    // no process
+                }
+                //Comment line
+                else if (line.charAt(0) == ';') {
+                    // no process
+                }
+                //Section line
+                else if (line.charAt(0) == '[' && line.charAt(line.length() - 1) == ']') {
+                    section = line.substring(1, line.length() - 1);
+                    map = new HashMap<>();
+                }
+                //Parameter line
+                else if (line.length() >= 3 && line.contains("=") && line.length() > line.indexOf("=") + 1) {
+                    String key = line.substring(0, line.indexOf("="));
+                    String value = line.substring(line.indexOf("=") + 1);
```

Comment:
I believe we can call indexOf only once. Just create variable.

Reply:
fixed in 6426040a3212c92a2da83afaa64b6a6eb29dc8c3